### ISSUE TITLE
Unnecessary nullity checks

### ISSUE THREAD

**Issue by TBotV63** [meguy26/RemoteConsole#5]
[`this != null`] will always be true, as if the current object was null, the method couldn't operate on it.

**Comment by meguy26** [comment]
True... some of that API stuff is just messy. SaveObjectSection is really ugly.
